### DISCUSSION TITLE
AWS registration problem

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1558] [Q&A]
sir when i tried to register for aws account then the payment method was shown like, payment method cannot be verified .  Now, What can i do sir, i tried this from 2 days but it will shows same issue

**Comment by sreedharm07** [comment]
some times some cards will not take by aws. change your card and try.

**Reply by Bhashitha19** [reply]
i had only the single card and that card will also not supported

**Reply by sreedharm07** [reply]
Cont do anything. if one card not works you should change the card.
